scripts/core/EXP-BINDLESS-IMAGES.rst

```diff
@@ -154,7 +154,7 @@ Changelog
 | 1.0      | Initial Draft                                           |
 +----------+---------------------------------------------------------+
 | 2.0      || Added device parameters to UR functions.               |
-|          || Added sub-region copy paramters to image copy function.|
+|          || Added sub-region copy parameters to image copy function.|
```

Comment:
this needs to be split across two lines now (assuming that's easier than fixing the table boundaries)

Reply:
well spotted, thx!